### DISCUSSION TITLE
RGB 데이터형과 처리 방식

### DISCUSSION THREAD

**Discussion by idealflower-k** [42seoul-jgo/miniRT#10] [General]
R,G,B colors in range [0,255]: 10, 0, 255

R, G, B의 범위만 명시 되어있을 뿐 자료형에 대한 언급은 없음.

1. RGB 값을 소수점으로 입력했을 때 (123.567) 소수부를 날리거나 반올림 해서 int형으로 처리하는 방법
2. RGB 값을 double 로 처리하는 방법
3. 소수점 입력을 에러 처리해서 종료하는 방법 (이거는 서브젝트에 int라고 명시가 없어서 디펜스가 힘들 수 도 있을 거 같음)

이중에 어느 방식이 렌더링하는데 도움이 될까요?

**Comment by JinmuGo** [comment]
음.... 3번은 확실히 아닌 것 같고 1번으로 반올림이나 내림을 해야할 것 같아요! 왜냐하면 애초에 rgb가 정수가 기본이어서 1번으로 처리하는게 맞다고 생각합니다!
